socorro/external/rabbitmq/crashstorage.py

```diff
@@ -326,3 +327,31 @@ def reprocess(self, crash_ids):
             ):
                 success = False
         return success
+
+
+#==============================================================================
+class PriorityjobRabbitMQCrashStore(RabbitMQCrashStorage):
+    required_config = Namespace()
+    required_config.rabbitmq_class = change_default(
+        RabbitMQCrashStorage,
+        'rabbitmq_class',
+        ConnectionContext,
+    )
+    required_config.add_option(
+        'routing_key',
+        default='socorro.priority',
+        doc='the name of the queue to recieve crashes',
+    )
```

Comment:
here, I tried to use that `change_default` function but it didn't work. It clashed with the `ReprocessOneRabbitMQCrashStore` class.